bin/appscale-upgrade.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+
+# General-purpose Python library imports
+import os
+import sys
+import traceback
+
+
+# AppScale library imports
+try:
+  import appscale
+  lib = os.path.dirname(appscale.__file__)
+except ImportError:
+  lib = os.path.dirname(__file__) + os.sep + ".." + os.sep + "lib"
+sys.path.append(lib)
+
+
+# Make sure we're on Python 2.6 or greater before importing any code
+# that's incompatible with older versions.
+import version_helper
+version_helper.ensure_valid_python_is_used()
+
+
+from appscale_logger import AppScaleLogger
+from appscale_tools import AppScaleTools
+from local_state import LocalState
+from parse_args import ParseArgs
```

Comment:
With the namespace package changes merged, you can do
```
from appscale.tools import version_helper
from appscale.tools.appscale_tools import AppScaleTools
```
etc.

You don't need the above try, except.